pyCGM_Single/tests/test_Pipelines.py

```diff
@@ -0,0 +1,517 @@
+import pytest
+import numpy as np
+from pyCGM_Single.pyCGM_Helpers import getfilenames
+from pyCGM_Single.pycgmIO import loadData, dataAsDict
+from pyCGM_Single.clusterCalc import target_dict, segment_dict
+from pyCGM_Single.pyCGM import pelvisJointCenter
+import pyCGM_Single.Pipelines as Pipelines
+import os
+
+rounding_precision = 8
+cwd = os.getcwd()
+if(cwd.split(os.sep)[-1]=="pyCGM_Single"):
+    parent = os.path.dirname(cwd)
+    os.chdir(parent)
+cwd = os.getcwd()
+
+def load_data_from_file(x, calcSacrum = False):
+    """
+    Loads motion capture data from dynamic and static trials.
+    Returns as data dictionaries. Optionally calculate the 
+    sacrum marker before creating the data dictionaries.
+    """
+    cur_dir = cwd
+    dynamic_trial, static_trial,_,_,_ = getfilenames(x)
+    dynamic_trial = os.path.join(cur_dir, dynamic_trial)
+    static_trial = os.path.join(cur_dir, static_trial)
+    motionData = loadData(dynamic_trial)
+    staticData = loadData(static_trial)
+    if (calcSacrum):
+        for frame in motionData:
+            frame['SACR'] = pelvisJointCenter(frame)[2]
+    data = dataAsDict(motionData, npArray=True)
+    static = dataAsDict(staticData, npArray=True)
+    return data, static
+
+"""
+The following functions test Pipelines.butterFilter(data, cutoff, Fs),
+where data is an array of numbers to filter, cutoff is the cutoff
+frequency to filter, and Fs is the sampling frequency of the data.
+
+We test cases where inputs are positive and negative floats and 
+integers. We test cases where inputs are lists and numpy arrays.
+
+We test exceptions raised when the length of the input data is too short, 
+when the cutoff frequency value is negative, zero, or too large, 
+and when the sampling frequency value is negative, zero, or too small.
+"""
+#Test positive and negative floats and ints
+listTests = [
```

Comment:
Please keep pep8 naming (except when its not possible because of pycgm)

Reply:
Changed variables to follow pep8 naming.